bin/api-docs/gen-components-docs/markdown/props.js

```diff
@@ -0,0 +1,53 @@
+function renderPropType( type ) {
+	switch ( type.name ) {
+		case 'enum': {
+			const MAX_ENUM_VALUES = 10;
```

Comment:
Should this const be declared up there at the module level?

Reply:
It feels a bit detached to me at the module level, but I moved it up to the function level.